scripts/performance/configs/llama/__init__.py

```
@@ -31,13 +31,15 @@
         llama3_70b_pretrain_config_gb200,
         llama3_70b_pretrain_config_gb300,
         llama3_70b_pretrain_config_h100,
+        llama3_70b_pretrain_config_vr200,
     )
     from .llama31_llm_pretrain import (
```

Flake8 (7.3.0)

[error] '.llama31_llm_pretrain.llama31_405b_pretrain_config_b200' imported but unused

(F401)